### PR TITLE
Add 'Inverse' payoff

### PR DESCRIPTION
**Pull request by arjun-io**
We may need this payoff if we want to flip the base and quote currency for certain price feeds

### CODE REVIEW

File: packages/perennial-oracle/test/unit/payoff/Inverse.test.ts [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+        ethers.utils.parseEther('-0.090909090909090909'),
+      )
+    })
+
+    it('modifies price per payoff (6)', async () => {
+      expect(await provider.payoff(ethers.utils.parseEther('-0.1'))).to.equal(ethers.utils.parseEther('-10'))
+    })
+
+    it('modifies price per payoff (7)', async () => {
+      await expect(provider.payoff(ethers.utils.parseEther('0'))).to.revertedWithPanic(0x12)
```
Comment (EdNoepel) [inline packages/perennial-oracle/test/unit/payoff/Inverse.test.ts:46]: Seems the most gas-efficient behavior.

**Review by EdNoepel — APPROVED** [review]
Do we need to test negative prices?  Not sure price is signed, honestly.

File: packages/perennial-oracle/contracts/payoff/Inverse.sol [hunk 2]
```diff
@@ -0,0 +1,11 @@
+// SPDX-License-Identifier: Apache-2.0
+pragma solidity 0.8.24;
+
+import "@openzeppelin/contracts/utils/math/Math.sol";
+import "../interfaces/IPayoffProvider.sol";
+
+contract Inverse is IPayoffProvider {
+    function payoff(Fixed18 price) external pure override returns (Fixed18) {
+        return Fixed18Lib.ONE.div(price);
```
Comment (EdNoepel) [inline packages/perennial-oracle/contracts/payoff/Inverse.sol:9]: `div` seems fine; I see no reason to round out.

**Review by kbrizzle — APPROVED** [review]
